network/transparent_vlan_endpointclient_linux.go

```diff
@@ -673,6 +675,17 @@ func (client *TransparentVlanEndpointClient) DeleteEndpointsImpl(ep *endpoint, _
 	return nil
 }
 
+// Creates a new retrier with a fixed delay, and treats all errors as retriable
+func (client *TransparentVlanEndpointClient) Retry(f func() error) error {
+	retrier := retry.Retrier{
+		Cooldown: retry.Max(numRetries, retry.Fixed(sleepDelay)),
+	}
+	return errors.Wrap(retrier.Do(context.Background(), func() error {
+		// we always want to retry, so all errors are temporary errors
+		return retry.WrapTemporaryError(f()) // nolint
```

Comment:
There are some errors that you've defined that I don't think quite fit the idea of a TemporaryError (one that could be resolved by retrying). For example, `errInvalidIPv4Address` is unlikely to be fixed without direct intervention, no?

You certainly can wrap everything and force everything to retry, but it's a blunt instrument.

Reply:
For transparent vlan, I wanted to preserve the existing behavior, which is all errors are considered retriable. I don't think `errInvalidIPv4Address` is used in transparent vlan client, or in the retry method.